tests/e2e/specs/cartDetailPage_spec.js

```diff
@@ -12,8 +12,8 @@ describe('CartDetailPage', () => {
     cy.addLineItem('/product/lemare-booties-0778-brown/M0E20000000E0XM', 2);
     cy.addLineItem('/product/lemare-booties-0778-grey/M0E20000000E0WX', 3);
     cy.visit('/cart');
-    cy.get('[data-test=cart-total-items]')
-      .contains(/^\s*5 items in total\s*$/);
+    cy.get('[data-test=cart-total-items]', { timeout: 20000 })
```

Comment:
Seeing there are so many `timeout: 20000` everywhere, what about having a global var with this timeout? This way, we can decide and adjust the milliseconds we want to use for this kind of timeout from a central place, without having to go through every file.

You can use a variable first, but you can even make it a env var in the future.

Reply:
Global variables don't seem to be possible here according to this issue:
https://github.com/cypress-io/cypress/issues/1121#issuecomment-353879959,
So far the best option were env variables but do you think that is necessary atm?

Reply:
No, actually I didn't mean that kind of global var. What I meant is to have a file where to declare the timeout and then export it / import it into the test.
And then there's the option of using Cypress.config(...) or Cypress.env(...) to obtain the value.
But from the 3 options, I cannot decide which one is better. So I leave it up to you.
And answering your question: yes, it is necessary. Having more than 10 times the same value spread across the tests is a sign that we have duplication. And duplication is difficult to maintain.
This PR is supposed to make maintainability easy, but you extract part of its benefits if you leave this duplication. Let's just make a great PR 💪